contrib/babelfishpg_tsql/Makefile

```diff
@@ -91,6 +94,17 @@ PG_CFLAGS += -g -Werror
 PG_CFLAGS += -fstack-protector-strong
 PG_CPPFLAGS += -I$(TSQLSRC) -I$(PG_SRC) -DFAULT_INJECTOR
 
+#preprocessor check for postgis file
+POSTGIS_LIB := $(pglib_dir)
+file_to_check := postgis-3.so
```

Comment:
Let's have only one variable as shown in the previous Makefile.

Reply:
Done